source/_integrations/hegel.markdown

```diff
@@ -0,0 +1,291 @@
+---
+title: Hegel Amplifier
+description: Instructions on integrating Hegel amplifiers into Home Assistant.
+ha_category:
+  - Media player
+ha_release: 2025.1
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@boazca'
+ha_domain: hegel
+ha_platforms:
+  - media_player
+ha_ssdp: true
+ha_quality_scale: silver
+ha_integration_type: device
+---
+
+The `hegel` integration allows you to control your [Hegel Music Systems](https://www.hegel.com/) amplifiers from Home Assistant. It uses Hegel's official IP control protocol over TCP and supports real-time push updates for a responsive experience.
+
+This integration provides complete control over your Hegel amplifier including power management, volume control, input selection, and mute functionality, all with instant feedback when changes are made via the front panel or remote control.
+
+## Prerequisites
+
+- Your Hegel amplifier must be connected to the same network as Home Assistant
+- The amplifier must support IP control (models H95, H120, H190, H190V, H390, H590, and Röst)
+- TCP port 50001 must be accessible between Home Assistant and your amplifier
+- UPnP/SSDP should be enabled on your network for automatic discovery
```

Comment:
```suggestion
- Your Hegel amplifier must be connected to the same network as Home Assistant.
- The amplifier must support IP control (models H95, H120, H190, H190V, H390, H590, and Röst).
- TCP port 50001 must be accessible between Home Assistant and your amplifier.
- UPnP/SSDP should be enabled on your network for automatic discovery.
```
[End sentences with a period](https://learn.microsoft.com/en-us/style-guide/punctuation/periods). Please apply throughout the page.